fdbserver/TSSMappingUtil.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * TSSMappingUtil.h
```

Comment:
Please rename this file to `TSSMappingUtil.actor.h` (you will also need the dirty `if-elif-endif` hack for include to work properly). With that change you will be able to write 

```
ACTOR Future<Void> readTSSMappingRYW(Reference<ReadYourWritesTransaction> tr, std::map<UID, StorageServerInterface>* tssMapping);
```

instead of
```
Future<Void> readTSSMappingRYW(Reference<ReadYourWritesTransaction> const& tr, std::map<UID, StorageServerInterface>* const& tssMapping);
```

This will not only make it more readable, tools like `clangd` will also be able to find the definition and declaration respectively.